tests/v0_tests/test_client.py

```diff
@@ -52,20 +52,20 @@ def test_error_handling_in_health_check(self):
                 self.assertIn("If you are trying to check the health on Marqo Cloud", cm.exception.message)
 
     def test_check_index_health_response(self):
-        self.test_index_name = self.create_test_index(self.generic_test_index_name)
-        res = self.client.index(self.test_index_name).health()
+        test_index_name = self.create_test_index(self.generic_test_index_name)
+        res = self.client.index(test_index_name).health()
         assert 'status' in res
         assert 'status' in res['backend']
 
     def test_check_index_health_query(self):
+        test_index_name = self.create_test_index(self.generic_test_index_name)
         with patch("marqo._httprequests.HttpRequests.get") as mock_get:
-            self.test_index_name = self.create_test_index(self.generic_test_index_name)
-            res = self.client.index(self.test_index_name).health()
+            res = self.client.index(test_index_name).health()
             args, kwargs = mock_get.call_args
-            self.assertIn(f"/{self.test_index_name}/health", kwargs["path"])
+            self.assertIn(f"health", kwargs["path"])
```

Comment:
I actually prefer having the full path (`/{test_index_name}/health`) checked

Reply:
I believe that it returns incorrect response with marqo 1.0.0 but I could be wrong

Reply:
@danyilq this was released in Marqo version 1.1.0 (which is only now being rolled onto the cloud. Please see [here](https://github.com/marqo-ai/marqo/releases/tag/1.1.0)